_summaries/glorot2011deep_1.md

```diff
@@ -1,19 +1,77 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{glorot2011deep}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ishank-arora
+score: 9
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+
+  * First paper (I believe) to prove using rectifier function ($$max(0,  x)$$) better than $$\textit{logistic sigmoid}$$ or $$\textit{hyperbolic tangent}$$ activation functions. Approaches this from the field of computational neuroscience. 
+
+  * Proposes the use of rectifying non-linearities as alternatives to the hyperbolic tangent or sigmoid in deep artificial neural networks, in addition to using an $$L_1$$ regularizer on the activation values to promote **sparsity** and prevent potential numerical problems with unbounded activation. 
+
+  * rectifier function brings together the fields of computational neuroscience and machine learning.
+
+  * Rectifying neurons are a better model of biological neurons than hyperbolic tangenet networks.
+
+  * There are gaps between computational neuroscience models and machine learning models. Two main gaps are:
```

Comment:
The three lines saying that the rectifier function brings together the fields of computational neuroscience and machine learning and that they are a better model of biological neurons and that there are gaps could probably be combined into one (e.g. "The rectifier function brings together the fields of computational neuroscience and machine learning since it is a better model of biological neurons than hyperbolic tangent networks. The main gaps were:")

Reply:
I agree, but I think it works better separated out. This is simply because I think repetition works better in a summary where the audience may not necessarily read the paper at all. Repetition really helps solidify the idea across about the how rectifier functions are better and how they bring the fields of machine learning and computational neuroscience together.